README.md

```diff
@@ -59,6 +57,8 @@ to the `dependencies` in your `component.json`.
 
 ## Building Fork Awesome
 
+⚠️ The Fork Awesome build process has been neglected for some time. The instructions below seem to work for modern Linux installations as far as building the fonts, but not building the site. One repeatable workaround is to follow the instructions using [Ubuntu 18.04](https://releases.ubuntu.com/18.04/ubuntu-18.04.5-desktop-amd64.iso), optionally within a [VirtualBox Virtual Machine](https://www.virtualbox.org/). We're working on it... ⚠️
```

Comment:
Do please try #312, because if that works, I would probably even remove the whole disclaimer!

As for ruby warnings: I'm running 2.5.0 through rbenv, and I stil get them :/ Better discuss in #311

Reply:
I am manually testing this against a variety of ruby versions using rvm :

```
   ruby-2.4.10 [ x86_64 ]
   ruby-2.5.1 [ x86_64 ]
   ruby-2.5.8 [ x86_64 ]
   ruby-2.6.0 [ x86_64 ]
   ruby-2.6.1 [ x86_64 ]
   ruby-2.6.2 [ x86_64 ]
   ruby-2.6.3 [ x86_64 ]
   ruby-2.6.4 [ x86_64 ]
   ruby-2.6.5 [ x86_64 ]
   ruby-2.6.6 [ x86_64 ]
   ruby-2.7.0 [ x86_64 ]
   ruby-2.7.2 [ x86_64 ]
```

Ruby versions < 2.6 are already end-of-life ( EoL ). But I am still testing them as they still might be in use in different environments. Ruby 2.5 went EoL only 4 months ago ( 2021-03-31 ).  And also Debian Buster has ruby 2.5.1.